src/store/modules/app.js

```diff
@@ -73,6 +84,46 @@ const getters = {
 };
 
 const actions = {
+  checkValidKeymap(_, keymap) {
+    if (
```

Comment:
you can make this simpler, just return the entire thing without the true false

Reply:
Like a one liner? not really a fan personnaly as there is several conditions. I think it's more readable like this.